docs/integrations/system_metrics.md

```diff
@@ -1,28 +1,68 @@
-By default, **Logfire** does not collect system metrics.
+The [`logfire.instrument_system_metrics()`][logfire.Logfire.instrument_system_metrics] method can be used to collect system metrics with **Logfire**, such as CPU and memory usage.
 
-To enable metrics, you need to install the `logfire[system-metrics]` extra:
+## Installation
+
+Install `logfire` with the `system-metrics` extra:
 
 {{ install_logfire(extras=['system-metrics']) }}
 
-### Available Metrics
```

Comment:
This section was not very informative, and is even less useful now that the full config dict is documented. In the future I'd like to add a section documenting each metric in more detail, but it's not a priority.